csv2rdf/csv2rdf_single_subject.py

```diff
@@ -1,3 +1,7 @@
+# Run this file in the command line.
+# Args = the input reconciled csv files using openrefine
+# Convert the CSV data to Turtle format
+# out_rdf.ttl can be safely imported into Virtuoso.
```

Comment:
```suggestion
"""
Run this file in the command line to convert csv files containing
reconciled data to turtle files.
The script takes an arbitrary number of positional arguments.
The first argument is a path to a relation mapping file for the
data in the csvs. The relations mapping file is a json file where the
keys are headers in the csv file and the values are URIs to Wikidata properties.
The following arguments are paths to reconciled csv files to be converted.
The script creates a file containing the turtle in the directory containing this
file called 'out_rdf.ttl'.
"""
```
Will you check if this seems correct. I made it a docstring, added more details, and corrected the description of the command line arguments.

Reply:
All is good except that the values are URIs to either Wikidata, Schema.org, or other used properties from the database websites (Wikidata is the better one).

Reply:
Ok, that makes sense. Can you update with that?

Reply:
Yes, it's updated